aptos-move/block-executor/src/executor.rs

```diff
@@ -1108,9 +1103,18 @@ where
                         &final_results,
                     ) {
                         if scheduler.halt() {
-                            let mut shared_commit_state_guard = shared_commit_state.acquire();
-                            let (_, maybe_error) = shared_commit_state_guard.dereference_mut();
-                            *maybe_error = Some(Error::FallbackToSequential(e));
+                            let mut shared_maybe_error = shared_maybe_error.acquire();
```

Comment:
This is my fault. I should have put a comment why shared_maybe_error can not be modified by any other thread here.

Reply:
added